### DISCUSSION TITLE
20220307

### DISCUSSION THREAD

**Discussion by tmiw** [ARSFI/paclink-open-source#9] [General]
This is a pre-production build intended to gauge current status of the project and determine what needs to be fixed. Brief testing indicates that packet (using Kenwood TM-D700A) and Telnet channels work; status of other channel types is TBD.

Contains (vs. last official Paclink release):

* SQLite-based data store for messages and other configuration information (stored in C:\Users\[username]\AppData\Roaming\Paclink).
* Conversion of original VB code to C#/.NET 6.0.

Instructions:

1. Unzip net6.0-windows.zip anywhere on your hard drive.
2. Run Paclink.exe inside the archive.
3. Reconfigure Paclink accordingly. (There is no import functionality from the existing Paclink at this time.)

Please leave comments in the release's discussion thread or email K6AQ@winlink.org with feedback. 👍 

<hr /><em>This discussion was created from the release <a href='https://github.com/ARSFI/paclink-open-source/releases/tag/nightly_20220307'>20220307</a>.</em>

**Comment by k0qed** [comment]
Browsing some articles about .NET 6 I came across Blazor, and then .NET MAUI and then Hybrid Blazor (Blazor and MAUI together) -- support is in Visual Studio 17.2 preview. Xamarin on steroids _and_ access to platform specific resources. 

Supports Android, IOS, Mac, and Windows Targets. No Linux support to start, but I saw several mentions of community efforts to provide it.

Exciting things!

**Reply by tmiw** [reply]
Indeed. We'll need to finish moving all of the remaining dialogs from the Paclink project to Paclink.UI.Common/Paclink.UI.Windows projects before we can start porting the UI to MAUI. Fortunately, once the port begins, we should be able to release something on Linux or Mac pretty quickly (e.g. by only supporting the minimum needed to get Telnet and packet working).
